custom/icds_reports/management/commands/backfill_supervisor_id.py

```diff
@@ -0,0 +1,66 @@
+# check all tables have non-empty awc_id
+ucrs_with_awc_id = [
+    # some awc_id of this UCR are null
+    'config_report_icds-cas_static-pregnant-tasks_cases_6c2a698f',
+    # some forms are from deleted locations, so loc table doesn't have data on some rows
+    'config_report_icds-cas_static-usage_forms_92fbe2aa',
+    # some awc_id of this UCR are null
+    'config_report_icds-cas_static-awc_mgt_forms_ad1b11f0'
```

Comment:
From what I can tell, these are expected to be filled out by LS. THis data source has a column called `location_id`. It may be easier to update the `supervisor_id` column with data from `location_id`

Reply:
if it already has the right data then there's no need to have an additional column, we can just use location_id. I must have missed that when I audited the data sources. Since that's already been rolled out is it safe to remove it from the DS now?

Reply:
No. removing a column from a data source causes a rebuild. (not sure if that's necessary, I forget if removiong columns requires table locking in postgres).

Althought htere should be few of these forms so a rebuild would be quick, but I think there would need to be more investigation before removing the column

Reply:
There are around 100k forms/rows for this UCR. I think it will be a quick rebuild, so it probably is okay if we simply remove `supervisor_id`. It's also easy to update our UCR rebuild logic to avoid a rebuild on removing columns and do a 'DROP column` instead.